cloud/services/loadbalancers.go

```diff
@@ -23,7 +23,7 @@ const (
 func CreateNodeBalancer(ctx context.Context, clusterScope *scope.ClusterScope, logger logr.Logger) (*linodego.NodeBalancer, error) {
 	var linodeNB *linodego.NodeBalancer
 
-	NBLabel := fmt.Sprintf("%s-api-server", clusterScope.LinodeCluster.Name)
```

Comment:
is the reason for this change so cluster names have the full 32 Chars for the nodebalancer name?

Reply:
Yes, it allows clusters to use the full 32-character limit for NodeBalancer labels: https://www.linode.com/docs/api/nodebalancers/#nodebalancer-create